net/tayga/src/opnsense/mvc/app/models/OPNsense/Tayga/General.xml

```diff
@@ -1,27 +1,31 @@
 <model>
     <mount>//OPNsense/tayga/general</mount>
     <description>Tayga configuration</description>
-    <version>0.0.2</version>
+    <version>0.0.3</version>
     <items>
         <enabled type="BooleanField">
             <default>0</default>
             <Required>Y</Required>
         </enabled>
         <v4address type="NetworkField">
             <default>192.168.255.1</default>
-            <Required>N</Required>
+            <Required>Y</Required>
         </v4address>
+        <v4destination type="NetworkField">
+            <default>192.168.254.1</default>
+            <Required>Y</Required>
+        </v4destination>
         <v6address type="NetworkField">
             <default>2001:db8:1::2</default>
-            <Required>N</Required>
+            <Required>Y</Required>
         </v6address>
         <v6prefix type="NetworkField">
             <default>2001:db8:1:ffff::/96</default>
```

Comment:
I suggest using `64:ff9b::/96` (RFC 6052) as the default. This would give you a working default configuration. `2001:db8::/32` is for documentation only and should never be used in the wild (RFC 3849).

Reply:
Then I'll change it, thx